README.md

```diff
@@ -1,13 +1,21 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The task was to use React to build a survey form.
+I have created the multi-step Reading Habits Survey with 5 questions by using different form input types (radio buttons, select dropdown, text input, checkbox). I have also included the summary of answers when the survey is submitted.
 
 ## The problem
 
```

Comment:
You did a very nice Readme here! Especially also for later, when people will check out your projects on Github. They will definitely get a good idea what the task was and how you approached it! 🥳  🙌